admin-auth/src/main/kotlin/com/epam/drill/admin/auth/service/CacheService.kt

```diff
@@ -0,0 +1,21 @@
+package com.epam.drill.admin.auth.service
+
+import io.ktor.auth.*
+
+/**
+ *  A service that manages the storage and retrieval of apiKeys.
+ */
+interface CacheService {
```

Comment:
If this service is only assotiated with ApiKey why not rename it into `ApiKeyCacheService` ?

Reply:
renamed